meta/documents/user_guide/en/updating.md

```diff
@@ -6,37 +6,61 @@ The Wayfair plugin may be updated at any time to ensure that it is providing the
 Prior to updating, review [the latest information on the Wayfair plugin's releases.](https://github.com/wayfair-contribs/plentymarkets-plugin/releases)
 
 ## Updating to the latest version of the Wayfair Plugin
-1. From the main Plentymarkets page, go to `Plugins` >> `Plugin set overview`
+1. From the main Plentymarkets page, go to `Plugins` >> `Plugin set overview`:
+
+    ![plugins_menu_plugin_set_overview](../../../images/en/plugins_menu_plugin_set_overview.png)
+
     * **Do not select the deprecated, similarly named `Plugin overview` option, if it appears in the menu.**

```

Comment:
I would move this line up, below line 9. Better to instruct people what NOT to do beforehand if they are following this guide step-by-step